apps/package.json

```diff
@@ -139,12 +117,14 @@
     "node-sass": "1.2.2",
     "npm-which": "2.0.0",
     "phantomjs-prebuilt": "^2.1.3",
-    "playground-io": "bcjordan/playground-io#more-components",
+    "playground-io": "bcjordan/playground-io#tap-cap-touch",
```

Comment:
Yeah, pretty sure. Code-studio was using a different version than apps. @bcjordan can hopefully confirm that I ended up with the right one here.